expyfun/_externals/h5io/_h5io.py

```diff
@@ -0,0 +1,297 @@
+# -*- coding: utf-8 -*-
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import sys
+import tempfile
+from shutil import rmtree
+from os import path as op
+
+import numpy as np
+try:
+    from scipy import sparse
+except ImportError:
+    sparse = None
+
+# Adapted from six
+PY3 = sys.version_info[0] == 3
+text_type = str if PY3 else unicode  # noqa
+string_types = str if PY3 else basestring  # noqa
+
+
+##############################################################################
+# WRITING
+
+def _check_h5py():
+    """Helper to check if h5py is installed"""
+    try:
+        import h5py
+    except ImportError:
+        raise ImportError('the h5py module is required to use HDF5 I/O')
+    return h5py
+
+
+def _create_titled_group(root, key, title):
+    """Helper to create a titled group in h5py"""
+    out = root.create_group(key)
+    out.attrs['TITLE'] = title
+    return out
+
+
+def _create_titled_dataset(root, key, title, data, comp_kw=None):
+    """Helper to create a titled dataset in h5py"""
+    comp_kw = {} if comp_kw is None else comp_kw
+    out = root.create_dataset(key, data=data, **comp_kw)
+    out.attrs['TITLE'] = title
+    return out
+
+
+def write_hdf5(fname, data, overwrite=False, compression=4,
+               title='h5io'):
```

Comment:
ahh, no, I misread this line and thought you had changed the title. We can just do `externals/_h5io.py`, though. For some reason Alex didn't like it but I think it's a bit easier actually.